cpp/test/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ if(BUILD_PRIMS_TESTS)
       prims/add_sub_dev_scalar.cu
       prims/adjustedRandIndex.cu
       prims/binary_op.cu
+      prims/batched_matrix.cu
```

Comment:
please keep the files in alphabetical order.

Reply:
done